lldb/source/Commands/CommandObjectPlatform.cpp

```diff
@@ -1139,6 +1097,15 @@ class CommandObjectPlatformProcessLaunch : public CommandObjectParsed {
     m_arguments.push_back({run_arg_arg});
   }
 
+  void
+  HandleArgumentCompletion(CompletionRequest &request,
+                           OptionElementVector &opt_element_vector) override {
```

Comment:
Any way you could move this to the generic CommandObject implementation?

Reply:
That one caused a test failure when I tried the simple implementation, so it goes into the second round.